assets/src/edit-story/theme.js

```diff
@@ -17,6 +17,7 @@ const theme = {
 			v3: '#242A3B',
 			v4: '#2F3449',
 			v5: '#575D65',
+			v6: '#07080C',
```

Comment:
I feel like at least one other PR has a `bg.v6` value (probably different from this) clearly illustrating the fragility of this scheme.

Reply:
Let's see who gets to merge first :)